cognite/neat/_graph/loaders/_rdf2dms.py

```diff
@@ -500,8 +473,6 @@ def _create_instances(
         properties: dict[str | InstanceType, list[Any]],
         projection: _Projection,
         stop_on_exception: Literal[True, False] = False,
-        exclude: set[str] | None = None,
-        include: set[str] | None = None,
```

Comment:
These were only used for hierarchical properties, thus removing them to simplify.